Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM maven:3.6.3-openjdk-8
+
+COPY . /usr/app/
```

Comment:
What do you do if you need to change code? rebuild the image, if yes where is the hot reload feature?